index.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
-<title>Improving the web without third-party cookies</title>
+<title>Third Party Cookies Must Be Deprecated</title>
```

Comment:
The word "deprecated" is a poor choice here.

"Deprecated" is a state where something is not removed, but discouraged.  This is not what we should be saying.

```suggestion
<title>Third Party Cookies Are Antithetical To Privacy</title>
```

```suggestion
<title>Third Party Cookies Are Incompatible With Privacy</title>
```


```suggestion
<title>Third Party Cookies Are Bad</title>
```

Reply:
How about:
```suggestion
<title>Third Party Cookies Considered Harmful</title>
```

Reply:
I prefer any of these four options to "Must Be Deprecated." Some thoughts:

Martin's three suggestions do not suggest a solution, they "merely" point out a problem. "Must Be Deprecated" was at least a call to action, if a call to the wrong specific action.

Considered Harmful is cliché, and it's not even *our* cliché, it's the IETF's. But as a term of art or whatever it does hit pretty hard to my ears. I wonder if it hits as hard as I think it does to an audience not as steeped in standards stuff.

I think the option that would go hardest would be something like:

```suggestion
<title>Third Party Cookies Are Incompatible With Privacy and Must Be Removed</title>
```

Reply:
I'm good with that.